package.json

```diff
@@ -36,28 +36,20 @@
   ],
   "scripts": {
     "changelog": "changelog-maker",
-    "test": "npm run clean && npm run browserify && npm run webpack && ava",
-    "test:browserify": "npm run browserify && ava test/browserify.spec.js",
-    "test:webpack": "npm run webpack && ava test/webpack.spec.js",
```

Comment:
I removed these because I didn't use them with the Cypress branch and also because I didn't find them as useful anymore now that tests run fast and don't error. I tend to use `npx jest ...` for custom usage anyway. Probably could mention in commit message but nbd tbh